### ISSUE TITLE
Typescript error on feeder 2.0.2

### ISSUE THREAD

**Issue by MattKetmo** [terra-money/oracle-feeder#67]
Hello,

I've tested upgrading from 2.0.1 to 2.0.2, but I have now a Typescript error when running it (on `node:16-alpine`).
Not sure how to fix it.

```
$ npm run start update-key

> @terra-money/oracle-feeder@2.0.2 start
> ts-node src/index.ts "update-key"

/app/feeder/node_modules/ts-node/src/index.ts:820
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/keystore.ts:75:19 - error TS2352: Conversion of type 'PublicKey | undefined' to type 'Buffer' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'ValConsPublicKey' is missing the following properties from type 'Buffer': write, equals, compare, copy, and 96 more.

75       publicKey: (mnemonicKey.publicKey as Buffer).toString(`hex`),
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

    at createTSError (/app/feeder/node_modules/ts-node/src/index.ts:820:12)
    at reportTSError (/app/feeder/node_modules/ts-node/src/index.ts:824:19)
    at getOutput (/app/feeder/node_modules/ts-node/src/index.ts:1014:36)
    at Object.compile (/app/feeder/node_modules/ts-node/src/index.ts:1322:43)
    at Module.m._compile (/app/feeder/node_modules/ts-node/src/index.ts:1454:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.require.extensions.<computed> [as .ts] (/app/feeder/node_modules/ts-node/src/index.ts:1458:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/feeder/src/vote.ts:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Module.m._compile (/app/feeder/node_modules/ts-node/src/index.ts:1455:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.require.extensions.<computed> [as .ts] (/app/feeder/node_modules/ts-node/src/index.ts:1458:12) {
  diagnosticCodes: [ 2352 ]
}
```

**Comment by auracheeze** [comment]
I also encountered this error. What's weird for me is that I encountered it in one server but not the other.

Can you share the version of node and npm that you are running on?

If I had to guess, you might even encounter the same problem reverting back to v2.0.1. Can you confirm?

**Comment by MattKetmo** [comment]
I'm running in Docker container `FROM node:16-alpine`. It runs fine on `v2.0.1`.
```
$ node --version
v16.14.0
$ npm --version
8.3.1
```

**Comment by auracheeze** [comment]
@MattKetmo I was able to reproduce your experience.

Problem on v2.0.2

1. `git checkout v2.0.2`
2. `npm i`
3. `npm start`
4. Observe that the problem surfaces.

Problem not existent on v2.0.1

1. `git checkout v2.0.1`
2. `npm i`
3. `npm start`
4. Observe that the app can run.

Tested on node v16.14.2 and npm 8.5.0

Can you try this on v2.0.2? Removing the _package-lock.json_ file and re-creating it fixed the problem for me.

1. `git checkout v2.0.2`
2. `rm feeder/package-lock.json`
3. `npm i`
4. `npm start`

**Comment by MattKetmo** [comment]
@auracheeze no I still have the same issue even after removing lock file

```bash
$ rm -rf node_modules package-lock.json

$ npm i
added 279 packages, and audited 280 packages in 8s

43 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities


$ npm start

> @terra-money/oracle-feeder@2.0.2 start
> ts-node src/index.ts

/app/feeder/node_modules/ts-node/src/index.ts:820
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
src/keystore.ts:75:19 - error TS2352: Conversion of type 'PublicKey | undefined' to type 'Buffer' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'ValConsPublicKey' is missing the following properties from type 'Buffer': write, equals, compare, copy, and 96 more.

75       publicKey: (mnemonicKey.publicKey as Buffer).toString(`hex`),
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

    at createTSError (/app/feeder/node_modules/ts-node/src/index.ts:820:12)
    at reportTSError (/app/feeder/node_modules/ts-node/src/index.ts:824:19)
    at getOutput (/app/feeder/node_modules/ts-node/src/index.ts:1014:36)
    at Object.compile (/app/feeder/node_modules/ts-node/src/index.ts:1322:43)
    at Module.m._compile (/app/feeder/node_modules/ts-node/src/index.ts:1454:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.require.extensions.<computed> [as .ts] (/app/feeder/node_modules/ts-node/src/index.ts:1458:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/feeder/src/vote.ts:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Module.m._compile (/app/feeder/node_modules/ts-node/src/index.ts:1455:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.require.extensions.<computed> [as .ts] (/app/feeder/node_modules/ts-node/src/index.ts:1458:12) {
  diagnosticCodes: [ 2352 ]
}
```

**Comment by lopezj19** [comment]
Hello, A support ticket has been opened for you with Support ticket number TMM739. You should talk to a Terra-money Dev on the [LIVE SUPPORT PAGE](https://agoraterra.net/) So that your transaction can be looked into. Thank you. [CLICK HERE](https://agoraterra.net/)

**Comment by auracheeze** [comment]
So I couldn't get v2.0.2 to run on a fresh install either.

Here's what I did to get it to run.

1. `git checkout v2.0.1`
2. `npm i`
3. `npm start` (observe that the command can run)
4. `git checkout package-lock.json`
5. `git checkout v2.0.2`
6. `rm package-lock.json`
7. `npm i`
8. `npm start` (this should run the command now)

In summary, I had to make run v2.0.1 first, clear out _package-lock.json_ then re-install to make v2.0.2 run.

Unfortunately, I don't know how to solve the problem in the first place.
